src/main/java/uk/gov/justice/laa/crime/cfecrime/utils/FullMeansTestOutcome.java

```diff
@@ -0,0 +1,83 @@
+package uk.gov.justice.laa.crime.cfecrime.utils;
+
+import lombok.AccessLevel;
+import lombok.extern.slf4j.Slf4j;
+import lombok.Getter;
+import uk.gov.justice.laa.crime.cfecrime.cma.response.AssessmentResult;
+import uk.gov.justice.laa.crime.cfecrime.enums.CaseType;
+import uk.gov.justice.laa.crime.cfecrime.enums.Outcome;
+@Getter
+@Slf4j
+public class FullMeansTestOutcome {
```

Comment:
Perhaps it's just me, but I find it a bit confusing having the 'outcome' as both the name of the class and its output. How about we call this class `FullMeansTestOutcomeCalculator` or ``FullMeansTestOutcomeLogic`?

Reply:
Now called ullMeansTestOutcomeCalculator